module/x/gravity/types/msgs.go

```diff
@@ -140,17 +141,25 @@ func (msg MsgSendToEth) ValidateBasic() error {
 		return sdkerrors.Wrap(sdkerrors.ErrInvalidAddress, msg.Sender)
 	}
 
-	// fee and send must be of the same denom
+	// bridge fee and send must be of the same denom
 	if msg.Amount.Denom != msg.BridgeFee.Denom {
 		return sdkerrors.Wrap(sdkerrors.ErrInvalidCoins,
-			fmt.Sprintf("fee and amount must be the same type %s != %s", msg.Amount.Denom, msg.BridgeFee.Denom))
+			fmt.Sprintf("bridge fee and amount must be the same type %s != %s", msg.Amount.Denom, msg.BridgeFee.Denom))
+	}
+	// chain fee and send must be of the same denom
```

Comment:
would be good to clarify here that bridge fee goes to Ethereum relayers and chain fee goes to stakers. Just so anyone looking at this code knows.

Reply:
Done